src/render/VolumeRaycastShader.cpp

```diff
@@ -31,217 +31,214 @@
 
 #include "render/VolumeRaycastShader.h"
 
-
 #include <glm/gtc/type_ptr.hpp>
 
-VolumeRaycastShader::VolumeRaycastShader() : m_use_blending{ false }, m_blend_volume{ 0 }, m_blending_alpha{ 0 }, m_clip_min{ 0.0 }, m_clip_max{ 1.0 }
-//: m_threshold{ 0.0f }, m_multiplier{ 0.5 }
+VolumeRaycastShader::VolumeRaycastShader() : m_use_blending{false}, m_blend_volume{0}, m_blending_alpha{0}, m_clip_min{0.0}, m_clip_max{1.0}
 {
 
-  /*m_viewport[0] = 1.0;
-  m_viewport[1] = 1.0;*/
-
   m_shader = "VolumeRaycastShader";
 
   m_vertexShader = "#version 330 core\n"
```

Comment:
I would move this into it's own `.vert` file

Reply:
https://github.com/brown-ccv/VR-Volumeviewer/issues/53